x-pack/filebeat/input/netflow/convert.go

```diff
@@ -65,7 +65,7 @@ func toBeatEventCommon(flow record.Record) (event beat.Event) {
 
 	// ECS Fields -- event
 	ecsEvent := common.MapStr{
-		"created":  flow.Timestamp,
+		"created":  time.Now(),
```

Comment:
@adriansr @ebeahan What do you think about this change? The current `flow.Timestamp` is the time when the router emitted the record (distinct from the actual flow start/end times).

> event.created contains the date/time when the event was first read by an agent, or by your pipeline.

This change would make the data consistent with other sources. On the other hand the time being used is the time when the event was created, except it's by the observer rather than agent.

Reply:
Yes, this makes sense. Its clear that it wasn't following the ECS convention. Also currently we were duplicating this same information in `@timestamp`.

> This field is distinct from `@timestamp` in that `@timestamp` typically contain the time extracted from the original event.
>
> In most situations, these two timestamps will be slightly different. The difference can be used to calculate the delay between your source generating an event, and the time when your agent first processed it. This can be used to monitor your agent’s or pipeline’s ability to keep up with your event source.

This change will allow to perform the above calculation, so LGTM.

Reply:
LGTM as well